### ISSUE TITLE
Runtime patch

### ISSUE THREAD

**Issue by paddy-r** [Leeds-MRG/Minos#152]
To patch ```vivarium.framework.randomness``` at runtime to avoid modifying file (as is currently done in makefile), although would need to apply patch every time. Default is string replacement, but can pass any function + arguments.

Usage is:
```
vivarium.framework.randomness = utils.patch(vivarium.framework.randomness, replacements={"(IndexError, TypeError)": "(IndexError, TypeError, KeyError)"})
```

Will add to ```utils``` for now, but can be moved somewhere else.

Possibly related to #19.

**Comment by paddy-r** [comment]
Fixed/closed with #153.

**Comment by paddy-r** [comment]
Reopening this to deal with issue encountered by Joao. If not actually related to this, will open new issue...
